src/Components/Modal/ModalDialog.tsx

```diff
@@ -64,26 +64,10 @@ const StyledSans = styled(Sans)`
   transition: color 0.14s ease;
   cursor: pointer;
   color: ${color("purple100")};
-  ${({ secondary }: { secondary: boolean }) =>
```

Comment:
This secondary style wasn't being used, and I think it was leftover from an old non-design-system design anyway.